integration-libs/cpq-quote/README.md

```diff
@@ -0,0 +1 @@
+This library provides Discount percentage to Spartacus UI. 
```

Comment:
MAJOR: please add more description into this. Please use readme files in other libraries as reference.